src/store/sagas/bonds.ts

```diff
@@ -84,7 +85,19 @@ export function* handleGetUserVested() {
 }
 
 export function* handleBuyBondWithWSOL(data: BuyBond) {
```

Comment:
Just for your information: currently, bounds are not used in production. They probably won't be used in the near future either. The method of fundraising in crypto has changed, so there is no need to maintain this feature as it was. However, if it has already been updated, that's good.